src/translations/en.json

```diff
@@ -216,7 +216,10 @@
         "repeat_set": "Repeat mode",
         "shuffle_set": "Shuffle",
         "text_to_speak": "Text to speak",
-        "nothing_playing": "Nothing playing"
+        "nothing_playing": "Nothing playing",
+        "group": "Group",
+        "group_players": "Group players",
```

Comment:
```suggestion
        "group_players": "Group media players",
```

Reply:
Should we clarify the key as well, or is it fine to just have `group_players` as it's in the `media_player` object?

Reply:
You can change it if you want since it's a new key, otherwise we prefer to keep it since the old translation is then kept for the translators to iterate on.